source/blog/2022-11-17-solidus-quarterly-roundup-q3-2022.html.markdown

```diff
@@ -0,0 +1,94 @@
+
+
+# What’s new
+
+
+# Solidus 3.2 is out!
+
+We promised. We delivered. Last quarter, [we said](https://solidus.io/blog/2022/07/12/solidus-quarterly-roundup-q2-2022) that 3.2 was going to be released VERY soon, and we did it.
+
+The new version contains the changes we mentioned last quarter, in particular, a new event bus and the new starter frontend are here! \
+But in case you missed it, here is a brief summary of what this release is bringing.
+
+
+## New Event Bus
+
+Before the Event Bus, there were just two options to extend Solidus: writing your own custom service classes or, in extreme cases, monkey patching.
+
+The biggest issue they had in common was the tight coupling: for example, if some actions needed to happen when an order was placed, the calling code had to be aware of the context and be updated whenever changes to the flow had to be made.
+
+That is where we implemented the first version of Solidus’ Event Bus. With time and real-World usage, and by carefully listening to your feedback, we noticed some changes were needed: above all, it was hard to test and debug.
+
+And that’s why the new Event Bus is born: it brings better support for async subscribers, testability, observability, and many other features.
+
+The new Event Bus has been developed as a separate gem, [omnes](https://github.com/nebulab/omnes), which implements the pub-sub pattern. Check out the README to learn how to start using it!
+
+We also have an [upgrade guide](https://edgeguides.solidus.io/customization/subscribing-to-events#upgrading-from-the-legacy-event-system) from the legacy event system to Omnes.
+
+Note: while the legacy event system is still supported, it'll be discontinued and removed in Solidus 4, so our recommendation is to start planning the migration.
+
+
+## Starter Frontend
+
+The new starter frontend is here to make our beloved users’ lives easier.
+
+On fresh Solidus installations, this modern and easy-to-extend codebase will provide a more efficient bootstrapping experience.
+
+The biggest difference with the old frontend is that the new one does not come as a gem: it is in fact installed via a command that creates all the necessary files inside of the project instead.
+
+This new approach has the primary benefit of not requiring Deface for customizations, thus making developers’ lives easier.
+
+Plus, following the Rails ideology, the new Starter Frontend is opinionated enough to streamline potentially tough decisions for you, while maintaining all the freedom that makes Solidus a great choice for any brand in need of total flexibility. In its current state, the new frontend should be considered just as a starting point, though. In fact, we plan a lot of changes to make it more in line with Rails developers’ expectations: the good news is that we can change it quite easily because it is no more a dependency hence no risk of breaking any backward compatibility. 
```

Comment:
`Rails ideology` -> I'd call it `The Rails Doctrine`, which is actually a specific thing, and we could also link it: https://rubyonrails.org/doctrine

Reply:
Fixed, 9bfaf20